checker/src/org/checkerframework/checker/index/lowerbound/LowerBoundAnnotatedTypeFactory.java

```diff
@@ -148,34 +150,19 @@ public void addComputedTypeAnnotations(Tree tree, AnnotatedTypeMirror type, bool
         }
     }
 
+    /** Returns the value checker's annotated type factory. */
```

Comment:
Minor: we capitalize, an proper names, "Value Checker", "MinLen Checker" (5 lines down), etc.